packages/edit-widgets/src/components/sidebar/widget-areas.js

```diff
@@ -47,7 +47,9 @@ export default function WidgetAreas( { selectedWidgetAreaId } ) {
 			<div className="edit-widgets-widget-areas__top-container">
 				<BlockIcon icon={ blockDefault } />
 				<div>
-					<p>{ description }</p>
+					<p>
+						<RawHTML>{ description }</RawHTML>
```

Comment:
`<RawHTML>` will render a `<div>` element in client side, which is not compatible inside `<p>`. I would suggest to just use `dangerouslySetInnerHTML` for this.

Reply:
👍